### PR TITLE
[ADD] [9.0] purchase request procurement

### PR DESCRIPTION
**Pull request by AaronHForgeFlow**
# Purchase Request Procurement

This module introduces the possibility of creating Purchase Requests on the basis of Procurement Orders.

This is a port from v8 to v9 The module in v8 is here: https://github.com/OCA/purchase-workflow/tree/8.0/purchase_request_procurement

### CODE REVIEW

**Comment by AaronHForgeFlow** [comment]
cc @jbeficent

File: purchase_request_procurement/models/purchase_request.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+    _inherit = "purchase.request"
+
+    @api.multi
+    def unlink(self):
+        procurement_obj = self.env['procurement.order']
+        for req in self:
+            proc_ids = procurement_obj.search([('request_id', '=', req.id)])
+            if proc_ids:
+                raise UserError(_('You cannot delete purchase request that '
```
Comment (JonathanNEMRY) [inline purchase_request_procurement/models/purchase_request.py]: Maybe an added `ondelete=restrict` onto the field declaration?

File: purchase_request_procurement/README.rst [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+check there if your issue has already been reported. If you spotted it first,
+help us smashing it by providing a detailed and welcomed feedback.
+
+Credits
+=======
+
+Contributors
+------------
+
+* Eficent Business and IT Consulting Services, S.L. <contact@eficent.com>
```
Comment (adrienpeiffer) [inline purchase_request_procurement/README.rst]: Can you add all contributors ?

**Comment by adrienpeiffer** [comment]
A little remark on the README

**Comment by AaronHForgeFlow** [comment]
Thank you @adrienpeiffer I'll fix it in the other Purchase Request modules too

**Comment by adrienpeiffer** [comment]
:+1: (Code review & functional tests). Thanks !

**Comment by AaronHForgeFlow** [comment]
As purchase request is already merged: https://github.com/OCA/purchase-workflow/pull/259 Should I create a new branch with purchase request and this module?

**Comment by adrienpeiffer** [comment]
No, it's not necessary. You can rebase this branch from OCA/9.0

**Comment by AaronHForgeFlow** [comment]
Thank you @adrienpeiffer Would a merge be also correct?

**Comment by adrienpeiffer** [comment]
If you want to add changes in upstream in your current branch it's better to rebase your branch.

**Comment by AaronHForgeFlow** [comment]
All right, again, thank you @adrienpeiffer

**Comment by AaronHForgeFlow** [comment]
@adrienpeiffer We had issues rebasing, for reasons we are uncapable to understand. Thus we were forced to merge from upstream.

**Review by JonathanNEMRY — APPROVED** [review]

**Review by JordiBForgeFlow — APPROVED** [review]
:+1 technical and functional

**Pull request merged**